<!DOCTYPE html>
<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <title></title>
        <meta name="description" content="">
        <meta name="viewport" content="width=device-width">

        <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->

        <link rel="stylesheet" href="css/normalize.css">
        <link rel="stylesheet" href="css/main.css">
        
        <!-- Site specific -->
        <meta name="keywords" content="Yi Xing, University of Iowa, Internal Medicine, Genomics, Bioinformatics, Comparative Genomics, Microarray, Exon Array, microRNA, splicing, alternative splicing, pre-mRNA splicing, Iowa, UIowa, Huntington&#39;s disease, Evolution, Comparative Genomics">
		<meta name="description" content="Xing Laboratory of RNA Genomics and Bioinformatics.">
		<link type="text/css" rel="stylesheet" href="./css/global.css">
		<title>Xing Laboratory of RNA Genomics and Bioinformatics</title>
        <!-- END Site specific -->
        
        <script src="js/vendor/modernizr-2.6.2.min.js"></script>
    </head>
    <body onload="fixNavBar()">
        <!--[if lt IE 7]>
            <p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
        <![endif]-->

        <!-- Add your site or application content here -->
		<div class="container">
			<div><img src="./img/banner.jpg" alt="Xing Laboratory of RNA Genomics and Bioinformatics"></div>

		  <!-- navbar include -->
		  
			<div>
				<ul class="navbar" id="myNavBar" style="height: 40pt;">
					<li><a class="currentnav" href="index.html">RESEARCH</a></li>
					<li><a class="nav" href="people.html">PEOPLE</a></li>
					<li><a class="nav" href="publications.html">PUBLICATIONS</a></li>
					<li><a class="nav" href="software.html">SOFTWARE</a></li>
					<li><a class="nav" href="news.html">NEWS</a></li>
					<li><a class="nav" href="contact.html">CONTACT</a></li>
					<li><a class="nav" href="positions.html">POSITIONS</a></li>
					<li><a class="nav" id="photoId" href="photos.html">PHOTOS</a></li>
				</ul>
			</div>

		  <div class="rightcontainer"> <img src="./img/side.jpg" alt="" width="130" height="207" border="1"> </div>
		  <div class="content">
			<p align="left"><font size="4"><b>Welcome to the Xing Laboratory of RNA Genomics and Bioinformatics!</b></font> 
			<p align="left"><font size="2"><b>Current Research Topics:</b></font><br>
			  The long-term goal of our research is to elucidate how genomic and environmental variation of RNA regulatory networks impacts phenotypic traits and diseases. 
			  We are a hybrid computational and experimental lab. We combine genomic, bioinformatic, molecular, and network approaches to study mRNA processing and 
			  post-transcriptional gene regulation in mammalian cells. We develop novel computational and statistical methods for analysis of massive genome and transcriptome data. 
			  We integrate computational studies with high-throughput and experimental research to systematically investigate 
			  the variation and dynamics of RNA regulatory networks between species, within human populations, and in response to developmental and disease signals. 
		<br>
			</p>
			
			<p align="left"><font size="4"><b><a href="news.html">Recent News </a></b></font>
				<li>May 2017: Our R01 grant to study the variation and regulation of alternative splicing in mammalian transcriptomes is renewed by the NIH for four more years. <p></p></li>
				<li>February 2017: Dr. Xing joins the Scientific Advisory Committee of the <a href="http://www.cegs-cdre.org/" target="_blank">Center for Dynamic RNA Epitranscriptomes</a> at the University of Chicago. This is a Center for Excellence in Genomic Sciences (CEGS) funded by the National Human Genome Research Institute, with the main focus being to develop single base, quantitative, and sensitive sequencing methods for RNA modifications.<p></p></li>
				<li>January 2017: Our <a href="http://www.nature.com/nmeth/journal/v13/n8/full/nmeth.3898.html" target="_blank">m6A-LAIC-seq method</a> for m<sup>6</sup>A level and isoform characterization sequencing is highlighted by <em>Nature Methods</em> <a href="http://www.nature.com/nmeth/journal/v14/n1/full/nmeth.4142.html" target="_blank">"Method of the Year 2016"</a> review for <a href="http://www.nature.com/nmeth/journal/v14/n1/full/nmeth.4110.html" target="_blank">"Epitranscriptome Analysis"</a>.<p></p></li>
				<li>January 2017: Bioinformatics Ph.D. student Zijun Zhang was awarded a Keystone Symposia Future of Science Fund to present his work at the meeting "Omics strategies to study the proteome". <p></p></li>
				<li>September 2016: We received a research grant from the <a href="http://www.parkerici.org/" target="_blank">Parker Institute for Cancer Immunotherapy</a>. <p></p></li>
				<li>September 2016: We received an R01 grant from the NIH <a href="https://datascience.nih.gov/bd2k" target="_blank">Big Data to Knowledge (BD2K) initiative </a> to develop statistical methods and informatics pipelines for integrating transcriptomic and proteomic big data. <p></p></li>
				<li>July 2016: We received a National Institute of Mental Health (NIMH) R01 grant (with Drs. Douglas Black and Kathrin Plath) to study alternative splicing regulation in the mammalian brain. <p></p></li>
				<li>July 2016: Jinkai's work on m<sup>6</sup>A-LAIC-seq (m<sup>6</sup>A level and isoform characterization sequencing) is published by <em><a href="http://www.nature.com/nmeth/journal/v13/n8/full/nmeth.3898.html" target="_blank">Nature Methods</a></em>. See <a href="http://newsroom.ucla.edu/releases/scientists-develop-new-way-to-measure-important-chemical-modification-on-rna" target="_blank">UCLA news release</a>, <a href="https://www.genomeweb.com/sequencing/new-sequencing-approach-examines-specific-rna-epigenetic-modifications" target="_blank">GenomeWeb news story</a>, and <a href="http://www.nature.com/nmeth/journal/v13/n8/full/nmeth.3928.html" target="_news">News & Views by Nature Methods</a>.<p></p></li>
				<li>June 2016: Shihao's SURVIV paper is published by <em><a href="http://www.nature.com/articles/ncomms11548" target="_blank">Nature Communications</a></em>. SURVIV is a statistical method for survival analysis of mRNA isoform variation in cancer transcriptomes. See <a href="http://newsroom.ucla.edu/releases/using-big-data-scientists-discover-biomarkers-that-could-help-give-cancer-patients-better-survival-estimates" target="_blank">UCLA news release</a> and <a href="https://www.genomeweb.com/informatics/algorithm-predicts-cancer-survival-transcript-isoform-ratios" target="_blank">GenomeWeb news story</a>.<p></p></li>
				<li>January 2016: Lan's work on primate-specific, protein-coding Alu exons is published by <em><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0876-5" target="_blank">Genome Biology</a></em> (also see "Proteome diversification by genomic parasites", <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0875-6" target="_blank"> research highlight</a> by Eli Eisenberg). <p></p></li>
			</p></p>
		</div>
			<div class="footer">
			<div class="right">
			  <p>© 2007-2018 Xing Laboratory</p>
			</div>
			<p><strong>Xing Lab</strong>, Department of Microbiology, Immunology, & Molecular Genetics, UCLA </p>
		  </div>

		</div>
		
		<!-- Start of javascript libraries -->
        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.0/jquery.min.js"></script>
        <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.9.0.min.js"><\/script>')</script>
        <script src="js/plugins.js"></script>
        <script src="js/main.js"></script>
        <!-- End of javascript libraries -->
        
        <!-- Start of StatCounter Code -->
		<script type="text/javascript">
		var sc_project=3973057; 
		var sc_invisible=1; 
		var sc_partition=31; 
		var sc_click_stat=1; 
		var sc_security="1c6ea699"; 
		</script>

		<script type="text/javascript" src="./js/counter.js"></script><noscript>&lt;div class="statcounter"&gt;&lt;a href="http://www.statcounter.com/free_hit_counter.html" target="_blank"&gt;&lt;img class="statcounter" src="http://c.statcounter.com/3973057/0/1c6ea699/1/" alt="free web page hit counter" &gt;&lt;/a&gt;&lt;/div&gt;</noscript>
		<!-- End of StatCounter Code -->

    </body>
</html>
