<!DOCTYPE html>
<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <title></title>
        <meta name="description" content="">
        <meta name="viewport" content="width=device-width">

        <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->

        <link rel="stylesheet" href="css/normalize.css">
        <link rel="stylesheet" href="css/main.css">
        
        <!-- Site specific -->
        <!--
        <script type="text/javascript">
		function fixNavBar() {
			if ($.browser.webkit || ($.browser.msie && $.browser.version >=7.0 && $.browser.version < 8.0)) {
				$("#photoId").width($("#myNavBar").width() - 7*$("#photoId").width());
			}
		 }
		</script>
		-->
        <meta name="keywords" content="Yi Xing, University of Iowa, Internal Medicine, Genomics, Bioinformatics, Comparative Genomics, Microarray, Exon Array, microRNA, splicing, alternative splicing, pre-mRNA splicing, Iowa, UIowa, Huntington&#39;s disease, Evolution, Comparative Genomics">
		<meta name="description" content="Xing Laboratory of RNA Genomics and Bioinformatics.">
		<link type="text/css" rel="stylesheet" href="./css/global.css">
		<title>Xing Laboratory of RNA Genomics and Bioinformatics</title>
        <!-- END Site specific -->
        
        <script src="js/vendor/modernizr-2.6.2.min.js"></script>
    </head>
    <body onload="fixNavBar()">
        <!--[if lt IE 7]>
            <p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
        <![endif]-->

        <!-- Add your site or application content here -->
		<div class="container">
			<div><img src="./img/banner.jpg" alt="Xing Laboratory of RNA Genomics and Bioinformatics"></div>

		  <!-- navbar include -->
		  
			<div>
				<ul class="navbar" id="myNavBar" style="height: 40pt;">
					<li><a class="nav" href="index.html">RESEARCH</a></li>
					<li><a class="nav" href="people.html">PEOPLE</a></li>
					<li><a class="currentnav" href="publications.html">PUBLICATIONS</a></li>
					<li><a class="nav" href="software.html">SOFTWARE</a></li>
					<li><a class="nav" href="news.html">NEWS</a></li>
					<li><a class="nav" href="contact.html">CONTACT</a></li>
					<li><a class="nav" href="positions.html">POSITIONS</a></li>
					<li><a class="nav" id="photoId" href="photos.html">PHOTOS</a></li>
				</ul>
			</div>

		  <div class="rightcontainer"> <img src="./img/side.jpg" alt="" width="130" height="207" border="1"> </div>
		    <div class="content">
				<!-- Page Content Starts Here -->
				
				    <P align=left><FONT size=4><b>Publications</b></FONT></P>
    <!-- <table width="100%" border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>Submitted and In Preparation</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>56. <strong>Zhao K.</strong>, <strong>Lu ZX.</strong>, <strong>Park JW.</strong>, Zhou Q., <strong>Xing Y.</strong> (2013) Robust statistical model for regulatory variation of alternative splicing using RNA-Seq data, Under Review.</li>
          <p></p>
		  </div>        
		</td>
      </tr>
    </table> 
	<br> -->
	<table width="100%" border="0" cellpadding="0" cellspacing="0">
      <tr>
      <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2018</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      
	  <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		  <li>90. Qin Z., Stoilov P., Zhang XG., <strong>Xing Y.</strong> (2018) SEASTAR: Systematic Evaluation of Alternative Transcription Start Sites in RNA. <em>Nucleic Acids Research</em>, In Press. </li><p></p>
		  <li>89. <strong>Zhang Z.</strong>, <strong>Park E.</strong>, <strong>Lin L.</strong>, <strong>Xing Y.</strong> (2018) A panoramic view of RNA modifications: exploring new frontiers. <em>Genome Biology</em>, 19(1):11.</li><p></p>
		 <li>88. <strong>Park E.</strong>, <strong>Pan Z.</strong>, <strong>Zhang Z.</strong>, <strong>Lin L.</strong>, <strong>Xing Y.</strong> (2018) The expanding landscape of alternative splicing variation in human populations. <em>American Journal of Human Genetics</em>, 102(1):11-26. [<a href="http://www.cell.com/ajhg/fulltext/S0002-9297(17)30454-8" target="_blank">journal</a>]  </li><p></p>
		
		</div>        
		</td>
      </tr>
	</table>
	<table width="100%" border="0" cellpadding="0" cellspacing="0">
      <tr>
      <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2017</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      
	  <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		 
		<li>87. <strong>Zhang Z.</strong>, <strong>Xing Y.</strong> (2017) CLIP-seq analysis of multi-mapped reads discovers novel functional RNA regulatory sites in the human transcriptome. <em>Nucleic Acids Research</em>, 45(16):9260-9271. [<a href="https://doi.org/10.1093/nar/gkx646" target="_blank">journal</a>][<a href="https://github.com/Xinglab/CLAM" target="_blank">software</a>]</li><p></p>
		<li>86. Zhou H., Molinie B., Daneshvar K., Pondick JV., <strong>Wang J.</strong>, Van Wittenberghe NO., <strong>Xing Y.</strong>, Giallourakis CC., Mullen AC. (2017) Identification and characterization of m6A circular RNA epitranscriptomes. <em>Cell Reports</em>, 20(9):2262-2276. [<a href="http://www.cell.com/cell-reports/abstract/S2211-1247(17)31134-8" target="_blank">journal</a>]</li><p></p>
		<li>85. <strong>Park E.</strong>, Guo J., <strong>Shen S.</strong>, Demirdjian L., Wu YN., <strong>Lin L.</strong>, <strong>Xing Y.</strong> (2017) Population and allelic variation of A-to-I RNA editing in human transcriptomes. <em>Genome Biology</em>, 18:143. [<a href="https://doi.org/10.1186/s13059-017-1270-7" target="_blank">journal</a>]</li><p></p>
		<li>84. <strong>Wang J.</strong>, <strong>Pan Y.</strong>, <strong>Shen S.</strong>, <strong>Lin L.</strong>, <strong>Xing Y.</strong> (2017) rMATS-DVR: rMATS discovery of Differential Variants in RNA. <em>Bioinformatics</em>, 33(14):2216-2217. [<a href="https://doi.org/10.1093/bioinformatics/btx128" target="_blank">journal</a>][<a href="https://github.com/Xinglab/rMATS-DVR" target="_blank">software</a>]</li><p></p>
		
		</div>        
		</td>
      </tr>
	</table>
	<table width="100%" border="0" cellpadding="0" cellspacing="0">
      <tr>
      <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2016</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      
	  <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>83. <strong>Lin L.</strong>, <strong>Park JW.</strong>, Ramachandran S., <strong>Zhang Y.</strong>, <strong>Tseng YT.</strong>, <strong>Shen S.</strong>, Waldvogel H., Curtis M., Faull R., Troncoso J., Ross C., Davidson BL.<sup>+</sup>, <strong>Xing Y.</strong><sup>+</sup> (2016) Transcriptome sequencing reveals aberrant alternative splicing in Huntington's disease. <em>Human Molecular Genetics</em>, 25(16):3454-3466. (<sup>+</sup> joint corresponding authors). [<a href="https://doi.org/10.1093/hmg/ddw187" target="_blank">journal</a>]</li><p></p>
		<li>82. Molinie B.*, <strong>Wang J.</strong>*, Lim KS., Hillebrand R., <strong>Lu ZX.</strong>, Van Wittenberghe N., Howard BD., Daneshvar K., Mullen A., Dedon P., <strong>Xing Y.</strong><sup>+</sup>, Giallourakis CC.<sup>+</sup> (2016) m<sup>6</sup>A level and isoform characterization sequencing (m<sup>6</sup>A-LAIC-seq) reveals the census and complexity of the m<sup>6</sup>A epitranscriptome. <em>Nature Methods</em>, 13(8):692-8. (<sup>+</sup> joint corresponding authors; <sup>*</sup> joint first authors). [<a href="http://www.nature.com/nmeth/journal/v13/n8/full/nmeth.3898.html" target="_blank">journal</a>] [<a href="http://newsroom.ucla.edu/releases/scientists-develop-new-way-to-measure-important-chemical-modification-on-rna" target="_blank">UCLA news release</a>] [<a href="https://www.genomeweb.com/sequencing/new-sequencing-approach-examines-specific-rna-epigenetic-modifications" target="_blank">GenomeWeb news story</a>] [<a href="http://www.nature.com/nmeth/journal/v13/n8/full/nmeth.3928.html" target="_news">News & Views by Nature Methods</a>]</li>
          <p></p>
		 <li>81. Lam MP., Venkatraman V., <strong>Xing Y.</strong>, Lau E., Cao Q., Ng DC., Su AI., Ge J., Van Eyk JE., Ping P. (2016) Data-driven approach to determine popular proteins for targeted proteomics translation of six organ systems. <em>J Proteome Res</em>, 15(11):4126–4134. [<a href="http://dx.doi.org/10.1021/acs.jproteome.6b00095" target="_blank">journal</a>]</li><p></p>
		<li>80. <strong>Shen S.</strong>, <strong>Wang Y.</strong>, <strong>Wang C.</strong>, Wu YN., <strong>Xing Y.</strong> (2016) SURVIV: Survival Analysis of mRNA Isoform Variation. <em>Nature Communications</em>, 7:11548. [<a href="http://www.nature.com/articles/ncomms11548" target="_blank">journal</a>] [<a href="http://newsroom.ucla.edu/releases/using-big-data-scientists-discover-biomarkers-that-could-help-give-cancer-patients-better-survival-estimates" target="_blank">UCLA news release</a>] [<a href="https://www.genomeweb.com/informatics/algorithm-predicts-cancer-survival-transcript-isoform-ratios" target="_blank">GenomeWeb news story</a>]</li>
          <p></p>
		 <li>79. <strong>Park JW.</strong><sup>+</sup>, Chung S., Rouchka E., <strong>Tseng YT.</strong>, <strong>Xing Y.</strong><sup>+</sup> (2016) rMAPS: RNA map analysis and plotting server for alternative exon regulation. <em>Nucleic Acids Research</em>. 44(W1):W333-8. (<sup>+</sup> joint corresponding authors).[<a href="http://nar.oxfordjournals.org/content/44/W1/W333" target="_blank">journal</a>] [<a href="http://rmaps.cecsresearch.org/" target="_blank">rMAPS web server</a>]</li><p></p>
		<li>78. Yang Y.*, <strong>Park JW.</strong>*, Bebee TW., Warzecha CC., <strong>Guo Y.</strong>, Shang X., <strong>Xing Y.</strong><sup>+</sup>, Carstens RP.<sup>+</sup> (2016) Determination of a comprehensive alternative splicing regulatory network and the combinatorial regulation by key factors during the epithelial to mesenchymal transition. <em>Molecular and Cellular Biology</em>, 36(11):1704-1719. (<sup>+</sup> joint corresponding authors; <sup>*</sup> joint first authors) [<a href="http://dx.doi.org/10.1128/MCB.00019-16" target="_blank">journal</a>]</li>
          <p></p>
		<li>77. <strong>Lin L.</strong>, <strong>Jiang P.</strong>, <strong>Park JW.</strong>, <strong>Wang J.</strong>, <strong>Lu ZX.</strong>, Lam MP., Ping P., <strong>Xing Y.</strong> (2016) The contribution of Alu exons to the human proteome. <em>Genome Biology</em>, 17:15. [<a href="http://dx.doi.org/10.1186/s13059-016-0876-5" target="_blank">journal</a>] [Genome Biology Research Highlight: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0875-6" target="_blank">Proteome diversification by genomic parasites</a>]</li>
          <p></p>
		<li>76. Cieply B.*, <strong>Park JW.</strong>*, Nakauka-Ddamba A., Bebee TW., <strong>Guo Y.</strong>, Shang X., Lengner CJ., <strong>Xing Y.</strong><sup>+</sup>, Carstens RP.<sup>+</sup> (2016) Multiphasic and dynamic changes in alternative splicing during induction of pluripotency are coordinated by numerous RNA binding proteins. <em>Cell Reports</em>, 15, 1–9. (<sup>+</sup> joint corresponding authors; <sup>*</sup> joint first authors). [<a href="http://dx.doi.org/10.1016/j.celrep.2016.03.025" target="_blank">journal</a>]</li>
          <p></p>
		<li>75. Ji X.*, <strong>Park JW.</strong>*, <strong>Bahrami-Samani E.</strong>, <strong>Lin L.</strong>, Duncan-Lewis C., Pherribo G., <strong>Xing Y.</strong><sup>+</sup>, Liebhaber SA.<sup>+</sup> (2016) &alpha;CP binding to a cytosine-rich subset of polypyrimidine tracts drives a novel pathway of cassette exon splicing in the mammalian transcriptome. <em>Nucleic Acids Research</em>, 44 (5): 2283-2297. (<sup>+</sup> joint corresponding authors; <sup>*</sup> joint first authors). [<a href="http://dx.doi.org/10.1093/nar/gkw088" target="_blank">journal</a>]</li>
          <p></p> 
		<li>74. <strong>Katrib A.</strong>, Hsu W., Bui A., <strong>Xing Y.</strong> (2016) “RADIOTRANSCRIPTOMICS”: A synergy of imaging & transcriptomics in clinical assessment. <em>Quantitative Biology</em>, 4(1): 1–12. [<a href="http://dx.doi.org/10.1007/s40484-016-0061-6" target="_blank">journal</a>]</li>
          <p></p>
		<li>73. Damianov A., Ying Y., Lin CH., Lee JA., Tran D., Vashisht AA., <strong>Bahrami-Samani E.</strong>, <strong>Xing Y.</strong>, Martin KC., Wohlschlegel JA., Black DL. (2016) Rbfox splicing regulators are recruited to pre-mRNA as part of a novel multi-protein complex, LASR. <em>Cell</em>, 165(3): 606-619.</li>
		<p></p> 
		<li>72. Ruscetti M., Dadashian EL., Guo W., Quach B., Mulholland DJ., <strong>Park JW.</strong>, Tran LM., Kobayashi N., Bianchi-Frias D.,   <strong>Xing Y.</strong>, Nelson PS., Wu H. (2016) HDAC inhibition impedes epithelial–mesenchymal plasticity and suppresses metastatic, castration-resistant prostate cancer. <em>Oncogene</em>, 35(29):3781-95. [<a href="http://dx.doi.org/10.1038/onc.2015.444" target="_blank">journal</a>]</li>
           
          <p></p>  
		</div>        
		</td>
      </tr>
	</table>
	<table width="100%" border="0" cellpadding="0" cellspacing="0">
      <tr>
      <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2015</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      
	  <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>71. Lam MP., Venkatraman V., Cao Q., Wang D., Dincer TU., Lau E., Su AI., <strong>Xing Y.</strong>, Ge J., Ping P., Van Eyk JE. (2015) Prioritizing Proteomics Assay Development for Clinical Translation. <em>J Am Coll Cardiol.</em>, 14;66(2):202-4. [<a href="http://dx.doi.org/10.1016/j.jacc.2015.04.072" target="_blank">journal</a>]</li>
          <p></p>  
		<li>70. <strong>Stein S.</strong>*, <strong>Lu ZX.</strong>*, <strong>Bahrami-Samani E.</strong>, <strong>Park JW.</strong>, <strong>Xing Y.</strong> (2015) Discover hidden splicing variations by mapping personal transcriptomes to personal genomes. <em>Nucleic Acids Research</em>, 43 (22): 10612-10622. ( <sup>*</sup> joint first authors). [<a href="http://dx.doi.org/10.1093/nar/gkv1099" target="_blank">journal</a>] [<a href="https://github.com/Xinglab/rPGA" target="_blank">rPGA software</a>]</li>
          <p></p> 
		<li>69. Bebee TW.*, <strong>Park JW.</strong>*, Sheridan KI., Warzecha CC., Cieply BW., Rohacek AM., <strong>Xing Y.</strong><sup>+</sup>, Carstens RP.<sup>+</sup> (2015) The splicing regulators Esrp1 and Esrp2 direct an epithelial splicing program essential for mammalian development, <em>Elife</em>, 4:e08954.(<sup>+</sup> joint corresponding authors; <sup>*</sup> joint first authors). [<a href="http://dx.doi.org/10.7554/eLife.08954" target="_blank">journal</a>] </li>
          <p></p>  
		<li>68. <strong>Wang J.</strong>*, <strong>Lu ZX.</strong>*, <strong>Tokheim C.</strong>, <strong>Miller SE.</strong>, <strong>Xing Y.</strong> (2015) Species-specific exon loss in human transcriptomes, <em>Molecular Biology and Evolution</em>, 32(2):481-494. (<sup>*</sup> joint first authors). [<a href="http://dx.doi.org/10.1093/molbev/msu317" target="_blank">journal</a>]</li>
          <p></p>  
		<li>67. <strong>Wang J.</strong>*, Ma MCJ.*, Mennie AK.*, Pettus JM., Xu Y., <strong>Lin L.</strong>, Traxler MG., Jakoubek J., Atanur SS., Aitman TJ., <strong>Xing Y.</strong><sup>+</sup>, Kwitek AE.<sup>+</sup> (2015) Systems biology with high-throughput sequencing reveals genetic mechanisms underlying the Metabolic Syndrome in the Lyon Hypertensive Rat, <em>Circulation: Cardiovascular Genetics</em>, 8:316-326. (<sup>+</sup> joint corresponding authors; <sup>*</sup> joint first authors). [<a href="http://dx.doi.org/10.1161/CIRCGENETICS.114.000520" target="_blank">journal</a>]</li>
          <p></p>  
		<li>66. <strong>Lu ZX.</strong>*, Huang Q.*,<strong>Park JW.</strong>*, <strong>Shen S.</strong>, <strong>Lin L.</strong>, <strong>Tokheim C.</strong>, Henry MD.<sup>+</sup>, <strong>Xing Y.</strong><sup>+</sup> (2015) Transcriptome-wide Landscape of Pre-mRNA Alternative Splicing Associated with Metastatic Colonization, <em>Molecular Cancer Research</em>, 13(2):305-18. (<sup>+</sup> joint corresponding authors; <sup>*</sup> joint first authors). [<a href="http://dx.doi.org/10.1158/1541-7786.MCR-14-0366" target="_blank">journal</a>] [<a href="http://mcr.aacrjournals.org/content/13/2/209.full.pdf" target="_blank">highlight by MCR</a>]</li>
          <p></p>  
		</div>        
		</td>
      </tr>
	</table>
	<table width="100%" border="0" cellpadding="0" cellspacing="0">
      <tr>
      <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2014</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      
	  <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>65. <strong>Shen S.</strong>*, <strong>Park JW.</strong>*, <strong>Lu ZX.</strong>, <strong>Lin L.</strong>, Henry MD., Wu YN., Zhou Q., <strong>Xing Y.</strong> (2014) rMATS: Robust and Flexible Detection of Differential Alternative Splicing from Replicate RNA-Seq Data, <em>Proc. Natl. Acad. Sci. U.S.A</em>., 111(51):E5593-601. (* joint first authors). [<a href="http://dx.doi.org/10.1073/pnas.1419161111" target="_blank">journal</a>]</li>
          <p></p>  
		<li>64. Batista PJ.*, Molinie B.*, <strong>Wang J.</strong>*, Qu K., Zhang J., Li L., Bouley DM., Lujan E., Haddad B., Daneshvar K., Carter AC., Flynn RA., Zhou C., Lim KS., Dedon P., Wernig M., Mullen AC., <strong>Xing Y.</strong><sup>+</sup>, Giallourakis CC.<sup>+</sup>, Chang HY.<sup>+</sup> (2014) m6A RNA modification controls cell fate transition in mammalian embryonic stem cells. <em>Cell Stem Cell</em>, 15(6):707-19. (<sup>+</sup> joint corresponding authors; <sup>*</sup> joint first authors). [<a href="http://dx.doi.org/10.1016/j.stem.2014.09.019" target="_blank">journal</a>]</li>
          <p></p>  
		<li>63. Guo R.*, Zheng L.*, <strong>Park JW.</strong>, Lv R., Chen H., Jiao F., Xu W., Mu S., Wen H., Qiu J., Wang Z., Yang P., Wu F., Hui J., Fu X., Shi X., Shi Y., <strong>Xing Y.</strong><sup>+</sup>, Lan F.<sup>+</sup>, Shi Y.<sup>+</sup> (2014) BS69/ZMYND11 Reads and Connects Histone H3.3 Lysine 36 Trimethylation Decorated Chromatin to Regulated Pre-mRNA Processing. <em>Molecular Cell</em>, 56(2):298-310. (<sup>+</sup> joint corresponding authors; <sup>*</sup> joint first authors). [<a href="http://dx.doi.org/10.1016/j.molcel.2014.08.022" target="_blank">journal</a>]</li>
          <p></p>  
		<li>62. Lackford B., Yao C., Charles GM., Weng L., Zheng X., Choi EA., Xie X., <strong>Wan J.</strong>, <strong>Xing Y.</strong>, Freudenberg JM., Yang P., Jothi R., Hu G., Shi Y. (2014) Fip1 regulates mRNA alternative polyadenylation to promote stem cell self-renewal, <em>EMBO J.</em>, 33(8):878-889. [<a href="http://dx.doi.org/10.1002/embj.201386537" target="_blank">journal</a>]</li>
          <p></p>  
		<li>61. <strong>Tokheim C.</strong>, <strong>Park JW.</strong>, <strong>Xing Y.</strong> (2014) PrimerSeq: Design and Visualization of RT-PCR Primers for Alternative Splicing Using RNA-seq Data, <em>Genomics, Proteomics & Bioinformatics</em>, 12(2):105-109. [<a href="http://dx.doi.org/10.1016/j.gpb.2014.04.001" target="_blank">journal</a>]</li>
          <p></p>
		<li>60. Boudreau RL., <strong>Jiang P.</strong>, Gilmore BL., Spengler RM., Tirabassi R., Nelson JA., Ross CA., <strong>Xing Y.</strong><sup>+</sup>, Davidson BL.<sup>+</sup> (2014) Transcriptome-wide discovery of microRNA binding sites in human brain, <em>Neuron</em>, 81(2):294-305. (<sup>+</sup> joint corresponding authors). [<a href="http://dx.doi.org/10.1016/j.neuron.2013.10.062" target="_blank">journal</a>]</li>
          <p></p>
		<li>59. McLoughlin HS., <strong>Wan J.</strong>, Spengler RM., <strong>Xing Y.</strong>, Davidson BL. (2014) Human-specific microRNA regulation of FOXO1: implications for microRNA recognition element evolution, <em>Hum. Mol. Genet.</em>, 23(10):2593-2603. [<a href="http://dx.doi.org/10.1093/hmg/ddt655" target="_blank">journal</a>]</li>
          <p></p>
		</div>        
		</td>
      </tr>
	</table>
	<table width="100%" border="0" cellpadding="0" cellspacing="0">
      <tr>
      <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2013</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      
	  <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>58. Yao C., Choi EA., Weng L., Xie X., <strong>Wan J.</strong>, <strong>Xing Y.</strong>, Moresco JJ., Tu PG., Yates JR 3rd., Shi Y. (2013) Overlapping and distinct functions of CstF64 and CstF64τ in mammalian mRNA 3' processing. <em>RNA</em>, 19:1781-1790. [<a href="http://dx.doi.org/10.1261/rna.042317.113" target="_blank">journal</a>]</li>
          <p></p>  
		<li>57. Heinicke LA., Nabet B., <strong>Shen S.</strong>, <strong>Jiang P.</strong>, van Zalen S., Cieply B., Russell JE., <strong>Xing Y.</strong>, Carstens RP. (2013) The RNA Binding Protein RBM38 (RNPC1) Regulates Splicing during Late Erythroid Differentiation. <em>PLoS ONE</em>, 8(10): e78031. [<a href="http://dx.doi.org/10.1371/journal.pone.0078031" target="_blank">journal</a>]</li>
          <p></p>  
		<li>56. <strong>Zhao K.</strong>, <strong>Lu ZX.</strong>, <strong>Park JW.</strong>, Zhou Q., <strong>Xing Y.</strong> (2013) GLiMMPS: Robust statistical model for regulatory variation of alternative splicing using RNA-Seq data, <em>Genome Biology</em>, 14:R74. [<a href="http://dx.doi.org/10.1186/gb-2013-14-7-r74" target="_blank">journal</a>] [<a href="http://www.mimg.ucla.edu/faculty/xing/glimmps/" target="_blank">GLiMMPS software</a>]</li>
          <p></p>  
		<li>55. <strong>Park JW.</strong>, <strong>Tokheim C.</strong>, <strong>Shen S.</strong>, <strong>Xing Y.</strong> (2013) Identifying differential alternative splicing events from RNA sequencing data using RNASeq-MATS. <em>Methods in Molecular Biology: Deep Sequencing Data Analysis</em>, Invited Book Chapter, 1038:171-179. [<a href="http://dx.doi.org/10.1007/978-1-62703-514-9_10" target="_blank">book</a>] [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23872975" target="_blank">PubMed</a>]</li>
          <p></p>
		<li>54. Ramachandran S., Karp PH., Osterhaus SR., <strong>Jiang P.</strong>, Wohlford-Lenane C., Lennox KA., Jacobi AM., Praekh K., Rose SD., Behlke MA., <strong>Xing Y.</strong>, Welsh MJ., McCray Jr PB. (2013) Post-transcriptional Regulation of CFTR Expression and Function by MicroRNAs. <em>Am. J. Respir. Cell Mol. Biol.</em>, 49(4):544-551. [<a href="http://dx.doi.org/10.1165/rcmb.2012-0430OC" target="_blank">journal</a>]</li>
          <p></p>  
		<li>53. Ji X.*, <strong>Wan J.</strong>*, Vishnu M., <strong>Xing Y.</strong><sup>+</sup>, Liebhaber SA.<sup>+</sup> (2013) The Poly-C Binding Proteins, αCPs, Act as Global Regulators of Alternative Polyadenylation. <em>Molecular and Cellular Biology</em>, 33(13):2560-2573 (<sup>+</sup> joint corresponding authors; * joint first authors). [<a href="http://dx.doi.org/10.1128/MCB.01380-12" target="_blank">journal</a>]</li>
          <p></p>
		  </div>        
		</td>
      </tr>
	</table>	  
    <br>
    <table width="100%" border="0" cellpadding="0" cellspacing="0">
      <tr>
      <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2012</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>52. Yao C., Biesinger J., <strong>Wan J.</strong>, Weng L., Busch A., <strong>Xing Y.</strong>, Xie X., Shi Y. (2012) Transcriptome-wide analyses of CstF64-RNA interactions in global regulation of mRNA alternative polyadenylation. <em>Proc. Natl. Acad. Sci. U.S.A</em>., 109(46):18773-18778. [<a href="http://dx.doi.org/10.1073/pnas.1211101109" target="_blank">journal</a>]</li>
          <p></p>
		<li>51. <strong>Brown SJ.</strong>, Stoilov P.<sup>+</sup>, <strong>Xing Y.</strong><sup>+</sup> (2012) Chromatin and epigenetic regulation of pre-mRNA processing. <em>Human Molecular Genetics</em>, Invited Review, 21(R1):90-96 (<sup>+</sup> joint corresponding authors). [<a href="http://dx.doi.org/10.1093/hmg/dds353" target="_blank">journal</a>] </li>
          <p></p>
		<li>50. Ramachandran S., Karp P., <strong>Jiang P.</strong>, Ostedgaard LS., Walz AE., Fisher JT., Keshavjee S., Lennox KA., Jacobi AM., Rose SD., Behlke MA., Welsh MJ.<sup>+</sup>, <strong>Xing Y.</strong><sup>+</sup>, McCray PB.<sup>+</sup> (2012) A microRNA network regulates expression and biosynthesis of wild-type and &#916;F508 mutant cystic fibrosis transmembrane conductance regulator. <em>Proc. Natl. Acad. Sci. U.S.A</em>., 109(33):13362-13367 (<sup>+</sup> joint corresponding authors). [<a href="http://dx.doi.org/10.1073/pnas.1210906109" target="_blank">journal</a>] [<a href="http://now.uiowa.edu/2012/07/cystic-fibrosis-discovery" target="_blank">news release</a>]</li>
          <p></p>
		<li>49. Dittmar KA.*, <strong>Jiang P.</strong>*, <strong>Park JW.</strong>*, Amirikian K, <strong>Wan J.</strong>, <strong>Shen S.</strong>, <strong>Xing Y.</strong><sup>+</sup>, Carstens RP.<sup>+</sup> (2012) Genome-wide determination of a broad ESRP-regulated post-transcriptional network by high throughput sequencing. <em>Molecular and Cellular Biology</em>, 32(8):1468-82 (<sup>+</sup> joint corresponding authors; * joint first authors). [<a href="http://dx.doi.org/10.1128/MCB.06536-11" target="_blank">journal</a>] [<a href="http://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?study=SRP011008" target="_blank">dataset</a>]</li>
          <p></p>
           
		<li>48. Kim J.*, <strong>Zhao K.</strong>*, <strong>Jiang P.</strong>*, <strong>Lu ZX.</strong>, <strong>Wang J.</strong>, Murray JC.<sup>+</sup>, <strong>Xing Y.</strong><sup>+</sup> (2012) Transcriptome landscape of the human placenta. <em>BMC Genomics, </em>13:115 (<sup>+</sup> joint corresponding authors; * joint first authors). [<a href="http://dx.doi.org/10.1186/1471-2164-13-115" target="_blank">journal</a>] [<a href="http://www.mimg.ucla.edu/faculty/xing/placenta" target="_blank">dataset</a>]</li>
          <p></p>
		<li>47. <strong>Shen S.</strong>, <strong>Park JW.</strong>, Huang J, Dittmar KA, <strong>Lu ZX.</strong>, Zhou Q., Carstens RP., <strong>Xing Y.</strong> (2012) MATS: a Bayesian framework for flexible detection of differential alternative splicing from RNA-Seq data. <em>Nucleic Acids Research</em>, 40(8): e61. [<a href="http://dx.doi.org/10.1093/nar/gkr1291" target="_blank">journal</a>] [<a href="http://rnaseq-mats.sf.net/" target="_blank">MATS software</a>]</li>
          <p></p>
                   
        </div>        </td>
      </tr>
      
    </table>
    <br>
    <table width="100%" border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2011</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
          
		<li>46. <strong>Lu ZX</strong>., <strong>Jiang P</strong>., <strong>Xing Y.</strong> (2011). Genetic variation of pre-mRNA alternative splicing in human populations<em>. Wiley Interdisciplinary Reviews: RNA (WIREs RNA)</em>, 3(4):581-92. [<a href="http://dx.doi.org/10.1002/wrna.120" target="_blank">journal</a>] [<a href="Publications/WIRES-RNA-2011.pdf">pdf</a>]</li>
          <p></p>
		<li>45. Xu W., Seok J., Mindrinos MN., Schweitzer A., Jiang H., Wilhelmy J., Clark T., Kapur K., <strong>Xing Y.</strong>, Faham M., Storey J., Moldawer LL., Maier RV., Tompkins RG., Wong WH., Davis R., Xiao W., and the Inflammation and Host Response to Injury Large-Scale Collaborative Research Program. (2011) A human transcriptome array for high throughput clinical studies. <em>Proc. Natl. Acad. Sci. U.S.A</em>., 108(9):3707-3712. [<a href="http://dx.doi.org/10.1073/pnas.1019753108" target="_blank">journal</a>]</li> 
          <p></p>   
		<li>44. <strong>Shen S.</strong>*, <strong>Lin L.</strong>*, Cai JJ., <strong>Jiang P.</strong>, <strong>Kenkel EJ.</strong>, <strong>Stroik MR.</strong>, <strong>Sato S.</strong>, Davidson BL., <strong>Xing Y.</strong>  (2011) Widespread establishment and regulatory impact of Alu exons in human genes. <em>Proc. Natl. Acad. Sci. U.S.A</em>., 108(7):2837-2842 (* joint first authors). [<a href="http://dx.doi.org/10.1073/pnas.1012834108" target="_blank">journal</a>] [<a href="http://arstechnica.com/science/news/2011/02/transposable-elements-choose-which-genes-make-us-human.ars" target="_blank">news report</a>] [<a href="http://news-releases.uiowa.edu/2011/february/022811genome_evolution.html" target="_blank">UI news release</a>] [<a href="http://publications.nigms.nih.gov/biobeat/11-03-16/index.html#3" target="_blank">NIH Biomedical Beat</a>]</li> <p></p>   
		<li>43. <strong>Lu ZX.</strong>*, <strong>Jiang P.</strong>*, Cai JJ., <strong> Xing Y.</strong> (2011) Context-dependent robustness to 5' splice site polymorphisms in human populations. <em>Human Molecular Genetics</em>, 20:1084-1096. (* joint first authors). [<a href="http://dx.doi.org/10.1093/hmg/ddq553" target="_blank">journal</a>]</li>
          <p></p> 
		<li>42. Liu S.*, <strong>Lin L.</strong>*, <strong>Jiang P.</strong>, Wang D., <strong>Xing Y.</strong> (2011) A comparison of RNA-Seq and high-density exon array for detecting differential gene expression between closely related species. <em>Nucleic Acids Research</em>, 39(2):578-588. (* joint first authors). [<a href="http://dx.doi.org/10.1093/nar/gkq817" target="_blank">journal</a>] [<a href="http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?study=SRP003276" target="_blank">dataset</a>]</li>
          <p></p> 
                             
        </div>        </td>
      </tr>
      
    </table>
    <br>
  
    
    
     <table width="100%" border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2010</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">


             

		<li>41. Warzecha CC., <strong>Jiang P.</strong>, Amirikian, K., Dittmar, KA., Lu, H., <strong>Shen, S.</strong>, Guo, W., <strong>Xing Y.</strong>, Carstens RP. (2010) An ESRP-regulated splicing programme is abrogated during the epithelial-mesenchymal transition. <em>EMBO J</em>, 29:3286-3300. [<a href="http://dx.doi.org/10.1038/emboj.2010.195" target="_blank">journal</a>] [<a href="http://news.pennmedicine.org/news-bites/2010/11/splicing-master-switch-impairs-cell-motility-and-may-prevent-cancer-spread.html" target="_blank">news bite</a>]</li>
          <p></p>
		<li>40. <strong>Lin L.</strong>*, <strong>Shen S.</strong>*, <strong>Jiang P.</strong>, <strong>Sato S.</strong>,  Davidson BL., <strong>Xing Y.</strong> (2010) Evolution of alternative splicing in primate brain transcriptomes. <em>Human Molecular Genetics</em>, 19:2958-2973 (* joint first authors). [<a href="http://dx.doi.org/10.1093/hmg/ddq201" target="_blank">journal</a>] [<a href="Publications/HMG2010.pdf" target="_blank">pdf</a>]</li>
          <p></p>
		<li>39. Au KF., Jiang H., <strong>Lin L.</strong>, <strong>Xing Y.</strong>, Wong WH. (2010) Detection of splice junctions from paired-end RNA-seq data by SpliceMap. <em>Nucleic Acids Research</em>, 38:4570-4578. [<a href="http://dx.doi.org/10.1093/nar/gkq211" target="_blank">journal</a>] [<a href="http://www.genomeweb.com/blog/week-nucleic-acids-research-2" target="_blank">Genomeweb News</a>]</li>
          <p></p>
		<li>38. <strong>Shen S.</strong>, Warzecha CC., Carstens RP., <strong>Xing Y.</strong> (2010) MADS+: discovery of differential splicing events from Affymetrix exon junction array data. <em>Bioinformatics</em>, 26:268-269. [<a href="http://dx.doi.org/10.1093/bioinformatics/btp643" target="_blank">journal</a>] [<a href="Publications/Bioinformatics2010.pdf" target="_blank">pdf</a>] [<a href="http://www.medicine.uiowa.edu/Labs/Xing/MADSplus/" target="_blank">MADS+ software</a>] </li>
          <p></p>
		<li>37. Monteys AM., Spengler RM., <strong>Wan J.</strong>, Tecedor L., Lennox KA., <strong>Xing Y.</strong>, Davidson BL. (2010) Structure and activity of putative intronic miRNAs promoters. <em>RNA</em>, 16(3):495-505 [<a href="http://dx.doi.org/10.1261/rna.1731910" target="_blank">journal</a>]</li>
          <p></p>
		<li>36. Lin CL., Evans V., <strong>Shen S.</strong>, <strong>Xing Y.</strong>, Richter JD. (2010) The nuclear experience of CPEB: implications for RNA processing and translational control. <em>RNA</em>, 16(2):338-348 [<a href="http://dx.doi.org/10.1261/rna.1779810" target="_blank">journal</a>]</li>
          <p></p>  
                             
          </div>        </td>
      </tr>
      
    </table>
    <br>
    
    
     
  
     <table border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2009</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
          
		<li>35. Lu H., <strong>Lin L.</strong>, <strong>Sato S.</strong>, <strong>Xing Y.</strong>, Lee CJ. (2009) Predicting functional alternative splicing by measuring RNA selection pressure from multigenome alignments. <em>PLoS Computational Biology</em>, 5(12): e1000608. [<a href="http://dx.doi.org/10.1371/journal.pcbi.1000608" target="_blank">journal</a>] [<a href="Publications/PLOSCB2009.pdf" target="_blank">pdf</a>] [<a href="http://www.genomeweb.com/blog/week-plos-63" target="_blank">Genomeweb Daily News</a>]</li>
          <p></p>
		<li>34. Borchert GM., Gilmore BL., Spengler RM., <strong>Xing Y.</strong>, Lanier W., Bhattacharya D., Davidson BL. (2009) Adenosine deamination in human transcripts generates novel microRNA binding sites. <em>Human Molecular Genetics</em>. 18:4801-4807. [<a href="http://dx.doi.org/10.1093/hmg/ddp443" target="_blank">journal</a>] [<a href="Publications/HMG2009-ADAR-MicroRNA.pdf" target="_blank">pdf</a>]</li>
          <p></p>
		<li>33. Warzecha CC., <strong>Shen S.</strong>, <strong>Xing Y.</strong>, Carstens RP. (2009) The epithelial splicing factors ESRP1 and ESRP2 positively and negatively regulate diverse types of alternative splicing events. <em>RNA Biology</em>, 6(5):546-563. [<a href="http://dx.doi.org/10.4161/rna.6.5.9606" target="_blank">journal</a>] [<a href="Publications/RNABiology2009.pdf" target="_blank">pdf</a>]</li>
          <p></p>
		<li>32. <strong>Lin L.</strong>*, Liu S.*, <strong>Brockway H.</strong>, Seok J., <strong>Jiang P.</strong>, Wong WH., <strong>Xing Y.</strong> (2009) Using high-density exon arrays to profile gene expression in closely related species. <em>Nucleic Acids Research</em>, 12: e90 (* joint first authors). [<a href="http://dx.doi.org/10.1093/nar/gkp420" target="_blank">journal</a>] [<a href="Publications/NAR2009.pdf" target="_blank">pdf</a>] [<a href="http://www.medicine.uiowa.edu/Labs/Xing/Primate-microarray/" target="_blank">Supplemental Data</a>] [<a href="http://gluegrant1.stanford.edu/~junhee/JETTA/index.html" target="_blank">JETTA software</a>]</li>
          <p></p>
		<li>31. <strong>Lin L.</strong>, <strong>Jiang P.</strong>, <strong>Shen S.</strong>, <strong>Sato S.</strong>,  Davidson BL., <strong>Xing Y.</strong> (2009) Large-scale analysis of exonized mammalian-wide interspersed repeats (MIRs) in primate genomes. <em>Human Molecular Genetics</em>. 18: 2204-2214 [<a href="http://dx.doi.org/10.1093/hmg/ddp152" target="_blank">journal</a>] [<a href="Publications/HMG2009.pdf" target="_blank">pdf</a>]</li>
          <p></p>
		<li>30. Boudreau R., McBride J., Martins I., <strong>Shen S.</strong>, <strong>Xing Y.</strong>, Carter B.,  Davidson BL. (2009) Non-allele-specific silencing of mutant and wildtype huntingtin demonstrates therapeutic efficacy in Huntington's disease mice. <em>Molecular Therapy</em>. 17:1053-1063 [<a href="http://dx.doi.org/10.1038/mt.2009.17" target="_blank">journal</a>] [<a href="Publications/MT2009.pdf" target="_blank">pdf</a>]</li>
          <p></p>
          </div>        
        </td>
      </tr>
      
    </table>
    <br>
	 <table border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2008</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>29. Packer AN., <strong>Xing Y.</strong>, Harper SQ., Jones L., Davidson BL. (2008) The bi-functional microRNA miR-9/miR-9* regulates REST and CoREST and is down-regulated in Huntington's Disease. <em>Journal of Neuroscience</em>. 28: 14341-14346 [<a href="http://dx.doi.org/10.1523/JNEUROSCI.2390-08.2008" target="_blank">journal</a>] [<a href="Publications/JN-HD2008.pdf" target="_blank">pdf</a>] [<a href="http://www.jneurosci.org/cgi/content/full/28/53/i" target="_blank">This Week in the Journal of Neuroscience</a>].</li>
          <p></p>
		<li>28. <strong>Lin L.</strong>, <strong>Shen S.</strong>, <strong>Tye A.</strong>, Cai JJ., <strong>Jiang P.</strong>, Davidson BL., <strong>Xing Y.</strong> (2008) Diverse splicing patterns of exonized Alu elements in human tissues. <em>PLoS Genetics</em>. 4(10):e1000225. [<a href="http://dx.doi.org/10.1371/journal.pgen.1000225" target="_blank">journal</a>] [<a href="http://news-releases.uiowa.edu/2008/october/101708exons_dna.html" target="_blank">UI news release</a>] [<a href="http://www.genomeweb.com/blog/week-plos-10" target="_blank">GenomeWeb Daily News</a>] [<a href="http://www.nature.com/hdy/journal/vaop/ncurrent/full/hdy200962a.html" target="_blank">News and Commentary at Heredity</a>] [<a href="Publications/PLoSGenetics2008_Supplement.zip" target="_blank">Supplemental Data</a>]</li>
            <p></p>
		<li>27. Kapur K., Jiang H., <strong>Xing Y.</strong>, Wong WH. (2008) Cross-hybridization modeling on Affymetrix Exon Arrays. <em>Bioinformatics</em>. 24: 2887-2893 [<a href="http://biogibbs.stanford.edu/~kkapur/GeneBASE/" target="_blank">software</a>] [<a href="http://dx.doi.org/10.1093/bioinformatics/btn571" target="_blank">journal</a>] [<a href="Publications/BIOINFO2008.pdf" target="_blank">pdf</a>]</li>
            <p></p>
		<li>26. Roy M., Kim N., <strong>Xing Y.</strong>, Lee C. (2008) The effect of intron length on exon creation ratios during evolution of mammalian genomes. <em>RNA</em>. 14: 2261-2273 [<a href="http://dx.doi.org/10.1261/rna.1024908" target="_blank">journal</a>] [<a href="Publications/RNA_intronlength_2008.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		  	
		<li>25. <strong>Xing Y.</strong><sup>+</sup>, Stoilov P., Kapur K., Han A., Jiang H., Shen S., Black DL., Wong WH.<sup>+</sup> (2008) MADS: a New and Improved Method for Analysis of Differential Alternative Splicing by Exon-tiling Microarrays. <em>RNA</em>.14(8):1470-1479 (<sup>+</sup> joint corresponding authors). [<a href="http://dx.doi.org/10.1261/rna.1070208" target="_blank">journal</a>] [<a href="Publications/RNA2008.pdf" target="_blank">pdf</a>] [<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE11344" target="_blank">dataset</a>] [<a href="http://www.icts.uiowa.edu/~yxng/MADS/" target="_blank">software</a>] </li>
            <p></p>
		<li>24. Chiao E., Elezar M., <strong>Xing Y.</strong>, Xiong A., Kmet M., Millan M., Glenn J., Wong WH., Baker J. (2008) Isolation and Transcriptional Profiling of Purified Hepatic Cells Derived from Human Embryonic Stem Cells. <em>Stem Cells</em>, 26(8):2032-2041.. [<a href="http://dx.doi.org/10.1634/stemcells.2007-0964" target="_blank">journal</a>] [<a href="Publications/StemCell2008.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li>23. Cahoy J., Emery B., Kaushal A., Foo L., Zamanian J., Christopherson K., <strong>Xing Y.</strong>, Lubischer J., Krieg P., Krupenko S., Thompson W., Barres B. (2008) A Transcriptome Database for Astrocytes, Neurons, and Oligodendrocytes: A New Resource for Understanding Brain Development and Function. <em>Journal of Neuroscience</em>. 28(1):264-278. [<a href="http://dx.doi.org/10.1523/JNEUROSCI.4178-07.2008" target="_blank">journal</a>] [<a href="Publications/JN2008.pdf" target="_blank">pdf</a>] [<a href="http://www.jneurosci.org/cgi/content/full/28/1/i" target="_blank">This Week in the Journal of Neuroscience</a>] [<a href="http://www.f1000biology.com/article/id/1098895" target="_blank">faculty of 1000 Must Read</a>] </li>
            <p></p>
        </div>        </td>
      </tr>
      
    </table>
	 <br>
    <table border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2007</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>22. Calarco JA.*, <strong>Xing Y.</strong>*, Caceres M.*, Calarco JP., Xiao X., Pan Q., Lee C., Preuss T., Blencowe B. (2007) Global analysis of alternative splicing differences between humans and chimpanzees. <em>Genes and Development</em>. 21:2963-2975. (* joint first authors). [<a href="http://dx.doi.org/10.1101/gad.1606907" target="_blank">journal</a>] [<a href="Publications/GenesDev2007.pdf" target="_blank">pdf</a>] [<a href="http://www.cbc.ca/technology/story/2007/11/15/chimp-study.html" target="_blank">news report</a>] [<a href="Publications/Cell_LeadingEdge_2008.pdf">highlight by Cell</a>] </li>
            <p></p>
		<li>21. <strong>Xing Y.</strong><sup>+</sup>, Ouyang Z., Kapur K., Scott MP., Wong WH.<sup>+</sup> (2007) Assessing the conservation of mammalian gene expression using high-density Exon Arrays. <em>Molecular Biology and Evolution</em>. 24(6):1-3. (<sup>+</sup> joint corresponding authors). [<a href="http://dx.doi.org/10.1093/molbev/msm061" target="_blank">journal</a>] [<a href="Publications/MBE2007.pdf" target="_blank">pdf</a>] [<a href="http://biogibbs.stanford.edu/~yxing/MBE/" target="_blank">dataset</a>] </li>
            <p></p>
		<li>20. Kapur K., <strong>Xing Y.</strong>, Ouyang Z., Wong WH. (2007) Exon arrays provide accurate assessments of gene expression. <em>Genome Biology</em>. 8:R82. [<a href="http://dx.doi.org/10.1186/gb-2007-8-5-r82" target="_blank">journal</a>] [<a href="Publications/GB2007.pdf" target="_blank">pdf</a>] [<a href="http://biogibbs.stanford.edu/~kkapur/GeneBASE/" target="_blank">software</a>]</li>
            <p></p>
		<li>19. <strong>Xing Y.</strong> (2007) Genomic analysis of alternative splicing in cancers. <em>Frontiers in Biosciences</em>. 12:4034-4041. [<a href="http://dx.doi.org/10.2741/2369" target="_blank">journal</a>] [<a href="Publications/FrontierBiosci2007.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li> 18. Parker DS., Hsiao RL., <strong>Xing Y.</strong>, Resch A., Lee C. (2007) Solving the problem of trans-genomic query with alignment tables. <em>IEEE Transactions on Computational Biology and Bioinformatics</em>. 2007:1073. [<a href="Publications/TCBB2007.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li>17. Lee JA., <strong>Xing Y.</strong>, Nguyen D., Xie J., Lee C., Black DL. (2007) Depolarization and CaM Kinase IV Modulate NMDA Receptor Splicing through Two Essential RNA Elements. <em>PLoS Biology</em>. 5(2):e40. [<a href="http://dx.doi.org/10.1371/journal.pbio.0050040" target="_blank">journal</a>] [<a href="Publications/PLOS-Biology2007.pdf" target="_blank">pdf</a>] </li>
            <p></p>
        </div>        </td>
      </tr>
      
    </table>
    <br>
    <table border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2006</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>16. <strong>Xing Y.</strong>, Kapur K., Wong WH. (2006) Probe selection and expression index computation of Affymetrix Exon Arrays. <em>PLoS ONE</em>. 1(1):e88. [<a href="http://dx.doi.org/10.1371/journal.pone.0000088" target="_blank">journal</a>] [<a href="Publications/PLOS-One2006.pdf" target="_blank">pdf</a>] [<a href="http://biogibbs.stanford.edu/~kkapur/GeneBASE/" target="_blank">software</a>] </li>
            <p></p>
		<li>15. <strong>Xing Y.</strong>*<sup>+</sup>, Yu T.*, Wu Y., Roy M., Kim J., Lee C.<sup>+</sup> (2006) An expectation-maximization algorithm for probabilistic reconstruction of full-length isoforms from splice graphs. <em>Nucleic Acids Research</em>. 34:3150-3160 (<sup>+</sup> joint corresponding authors; * joint first authors). [<a href="http://dx.doi.org/10.1093/nar/gkl396" target="_blank">journal</a>] [<a href="Publications/NAR2006.pdf" target="_blank">pdf</a>] [<a href="http://www.bioinformatics.ucla.edu/yxing/isoform/" target="_blank">source code</a>] </li>
            <p></p>
		<li>14. <strong>Xing Y.</strong>, Wang Q., Lee C. (2006) Evolutionary divergence of exon flanks: A dissection of mutability and selection. <em>Genetics</em>. 173(3):1787-91. [<a href="http://dx.doi.org/10.1534/genetics.106.057919" target="_blank">journal</a>] [<a href="Publications/Genetics2006.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li>13. <strong>Xing Y.</strong>, Lee C. (2006) Alternative splicing and RNA selection pressure: evolutionary consequences for eukaryotic genomes. <em>Nature Reviews Genetics</em>. 7:499-509. [<a href="http://dx.doi.org/10.1038/nrg1896" target="_blank">journal</a>] [<a href="Publications/NRG2006.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li>12. <strong>Xing Y.</strong>, Lee C. (2006) Can RNA selection pressure distort the measurement of Ka/Ks? <em>Gene</em>. 370:1-5. [<a href="http://dx.doi.org/10.1016/j.gene.2005.12.015" target="_blank">journal</a>] [<a href="Publications/GENE2006.pdf" target="_blank">pdf</a>] </li>
            <p></p>
          </div>
        </td>
      </tr>
    </table>
    <br>
    <table border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2005</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>11. <strong>Xing Y.</strong>, Lee C. (2005) Protein modularity of alternatively spliced exons is associated with tissue-specific regulation of alternative splicing. <em>PLoS Genetics</em>. 1(3):e34. [<a href="http://dx.doi.org/10.1371/journal.pgen.0010034" target="_blank">journal</a>] [<a href="Publications/PLOS-Genetics2005.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li>10. <strong>Xing Y.</strong>, Lee C. (2005) Evidence of functional selection pressure for alternative splicing events that accelerate evolution of protein subsequences. <em>Proc. Natl. Acad. Sci. U.S.A</em>. 102:13526-13531.[<a href="http://dx.doi.org/10.1073/pnas.0501213102" target="_blank">journal</a>] [<a href="Publications/PNAS2005.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li>9. <strong>Xing Y.</strong>, Lee,C. (2005) Assessing the application of Ka/Ks ratio test to alternatively spliced exons. <em>Bioinformatics</em>. 21: 3701-3703. [<a href="http://dx.doi.org/10.1093/bioinformatics/bti613" target="_blank">journal</a>] [<a href="Publications/BIOINFO2005.pdf" target="_blank">pdf</a>] </li>
            <p></p>
          </div>
        </td>
      </tr>
    </table>
    <br>
    <table border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2004</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>8. Resch A., <strong>Xing Y.</strong>, Alekseyenko A., Modrek B., Lee C. (2004) Evidence for a subpopulation of conserved alternative splicing events under selection pressure for protein reading frame preservation. <em>Nucleic Acids Research</em>. 32:1261-9. [<a href="http://dx.doi.org/10.1093/nar/gkh284" target="_blank">journal</a>] [<a href="Publications/NAR2004.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li>7. <strong>Xing Y.</strong>, Lee C. (2004) Negative selection pressure against premature protein truncation is reduced by alternative splicing and diploidy. <em>Trends in Genetics</em>. 20:472-5. [<a href="http://dx.doi.org/10.1016/j.tig.2004.07.009" target="_blank">journal</a>] [<a href="Publications/TIG2004.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li>6. Resch A., <strong>Xing Y.</strong>, Modrek B., Gorlick M., Riley R., Lee C. (2004) Assessing the impact of alternative splicing on domain interactions in the human proteome. <em>Journal of Proteome Research</em>. 3:76-83. [<a href="http://dx.doi.org/10.1021/pr034064v" target="_blank">journal</a>] [<a href="Publications/JPR2004.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li>5. <strong>Xing Y.</strong>, Resch A., Lee C. (2004) The multiassembly problem: reconstructing multiple transcript isoforms from EST fragment mixtures. <em>Genome Research</em>. 14(3):426-4. [<a href="http://dx.doi.org/10.1101/gr.1304504" target="_blank">journal</a>] [<a href="Publications/GR2004.pdf" target="_blank">pdf</a>] [<a href="http://www.bioinformatics.ucla.edu/ASAP2/" target="_blank">database</a>] </li>
            <p></p>
		<li>4. Grasso C., Modrek B., <strong>Xing Y.</strong>, Lee C. (2004) Genome-wide detection of alternative splicing in expressed sequences using partial order multiple sequence alignment graphs. <em>Pac. Symp. Biocomputing. World Scientific: Singapore</em> 29-41. [<a href="Publications/PSB2004.pdf" target="_blank">pdf</a>] </li>
            <p></p>
          </div>
        </td>
      </tr>
    </table>
    <br>
    <table border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td width="50%" align="center" bgcolor="#FFCC99"><font size=+1>2003</font></td>
        <td width="90%">&nbsp;</td>
      </tr>
      <tr>
        <td colspan="2" bgcolor="#FFFFEE" style="border-style: dashed; border-width:1px; border-color: #FFCC99;">
          <div style="padding:10px;">
		<li>3. Zewail A.*, Xie MW.*, <strong>Xing Y.</strong>, Lin L., Zhang PF., Zou W., Saxe JP., Huang J. (2003) Novel functions of the phosphatidylinositol metabolic pathway discovered by a chemical genomics screen with wortmannin. <em>Proc. Natl. Acad. Sci. U.S.A</em>. 100: 3345-3350. [<a href="http://dx.doi.org/10.1073/pnas.0530118100" target="_blank">journal</a>] [<a href="Publications/PNAS2003.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li>2. <strong>Xing Y.</strong>, Xu Q., Lee C. (2003) Widespread production of novel soluble protein isoforms by alternative splicing removal of transmembrane anchoring domains. <em>FEBS Letters</em>. 555(3):572-8. [<a href="http://dx.doi.org/10.1016/S0014-5793(03)01354-1" target="_blank">journal</a>] [<a href="Publications/FEBS2003.pdf" target="_blank">pdf</a>] </li>
            <p></p>
		<li>1. Lee C., Atanelov L., Modrek B., <strong>Xing Y.</strong> (2003) ASAP: the Alternative Splicing Annotation Project. <em>Nucleic Acids Research</em>. 31(1):101-5. [<a href="http://dx.doi.org/10.1093/nar/gkg029" target="_blank">journal</a>] [<a href="Publications/NAR2003.pdf" target="_blank">pdf</a>] </li>
            <p></p>
          </div>
        </td>
      </tr>
    </table>
    <br>
    <br>
			
				<!-- Page Content Ends Here -->
		    </div>
			
			<div class="footer">
				<div class="right">
				  <p>© 2007-2018 Xing Laboratory</p>
				</div>
				<p><strong>Xing Lab</strong>, Department of Microbiology, Immunology, & Molecular Genetics, UCLA </p>
			</div>

		  </div>
		
		<!-- Start of javascript libraries -->
        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.0/jquery.min.js"></script>
        <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.9.0.min.js"><\/script>')</script>
        <script src="js/plugins.js"></script>
        <script src="js/main.js"></script>
        <!-- End of javascript libraries -->
        
        <!-- Start of StatCounter Code for Default Guide -->
<script type="text/javascript">
var sc_project=3973057; 
var sc_invisible=1; 
var sc_security="1c6ea699"; 
var scJsHost = (("https:" == document.location.protocol) ?
"https://secure." : "http://www.");
document.write("<sc"+"ript type='text/javascript' src='" +
scJsHost+
"statcounter.com/counter/counter.js'></"+"script>");
</script>
<noscript><div class="statcounter"><a title="Web Analytics
Made Easy - StatCounter" href="http://statcounter.com/"
target="_blank"><img class="statcounter"
src="//c.statcounter.com/3973057/0/1c6ea699/1/" alt="Web
Analytics Made Easy - StatCounter"></a></div></noscript>
<!-- End of StatCounter Code for Default Guide -->
    </body>
</html>
