<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Xing Laboratory of RNA Genomics and Bioinformatics by Xinglab</title>
    <link rel="stylesheet" href="../stylesheets/styles.css">
    <link rel="stylesheet" href="../stylesheets/github-dark.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
    <script src="../javascripts/main.js"></script>
    <!--[if lt IE 9]>
      <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">

  </head>
  <body>

    <header>
      <h1>Xing Laboratory of RNA Genomics and Bioinformatics</h1>
      <h2>Department of Microbiology, Immunology, & Molecular Genetics</h2>
      <h2>University of California, Los Angeles (UCLA)</h2>
    </header>

    <div id="banner">
      <span id="logo"></span>

      <a href="https://github.com/Xinglab" class="button fork"><strong>View On GitHub</strong></a>
    </div><!-- end banner -->

  <div class="wrapper">
    <nav>
        <ul><a href="/" class="button fork"><strong>Home</strong></a></ul>
        <ul><a href="/people/" class="button fork"><strong>People</strong></a></ul>
        <ul><a href="/publications/" class="button fork"><strong>Publications</strong></a></ul>
        <ul><a href="/software/" class="button fork"><strong>Software</strong></a></ul>
        <ul><a href="/news/" class="button fork"><strong>News</strong></a></ul>
        <ul><a href="/contact/" class="button fork"><strong>Contact</strong></a></ul>
        <ul><a href="/positions/" class="button fork"><strong>Positions</strong></a></ul>
        <ul><a href="/photos/" class="button fork"><strong>Photos</strong></a></ul>
      </nav>
      <section>

<h3>MATS:</h3>

<p>MATS is a computational tool to detect differential alternative splicing
events from RNA-Seq data. The statistical model of MATS calculates the P-value
and false discovery rate that the difference in the isoform ratio of a gene
between two conditions exceeds a given user-defined threshold. From the RNA-Seq
data, MATS can automatically detect and analyze alternative splicing events
corresponding to all major types of alternative splicing patterns. MATS handles
replicate RNA-Seq data from both paired and unpaired study design. </p>

<hr>

<h3>rMATS:</h3>
<p>a new statistical model and computer program, replicate MATS (rMATS),
designed for detection of differential alternative splicing from replicate
RNA-Seq data. rMATS uses a hierarchical model to simultaneously account for
sampling uncertainty in individual replicates and variability among replicates.
In addition to the analysis of unpaired replicates, rMATS also includes a model
specifically designed for paired replicates between sample groups. The
hypothesis-testing framework of rMATS is flexible and can assess the statistical
significance over any user-defined magnitude of splicing change. The performance
of rMATS is evaluated by the analysis of simulated and real RNA-Seq data. </p>

<hr>

<h3>rPGA:</h3>
<p>RNA-seq Personal Genome-alignment Analyzer (rPGA) is a pipeline to discover
hidden splicing variations by mapping personal transcriptomes to personal
genomes.</p>

<hr>


      </section> <footer> <p><small>Hosted on GitHub Pages &mdash; Theme by <a
      href="https://twitter.com/michigangraham">mattgraham</a></small></p>
      </footer> </div> <!--[if
      !IE]><script>fixScale(document);</script><![endif]-->

  </body>
</html>
